api/src/test/java/io/serverlessworkflow/api/test/MarkupToWorkflowTest.java

```diff
@@ -569,6 +569,23 @@ public void testConstants(String workflowLocation) {
     JsonNode serbianTranslationNode = translationDogNode.get("Serbian");
     assertEquals("pas", serbianTranslationNode.asText());
   }
+  
+  
+  @ParameterizedTest
+  @ValueSource(strings = {"/features/constantsRef.json", "/features/constantsRef.yml"})
+  public void testConstantsRef(String workflowLocation) {
+    Workflow workflow = Workflow.fromSource(WorkflowTestUtils.readWorkflowFile(workflowLocation));
+
+    assertNotNull(workflow);
+    assertNotNull(workflow.getId());
+    assertNotNull(workflow.getName());
+    assertNotNull(workflow.getStates());
+
+    assertNotNull(workflow.getConstants());
+    Constants constants = workflow.getConstants();
+    assertNotNull(constants.getRefValue());
```

Comment:
I think This assertion needs to be more strict to ensure the new behavior. recommend changing the assertion to assertEquals("  constantValues.json".constants.getRefValue());

Reply:
yes, better that way ;)